crypto/src/pbc/secure.rs

```diff
@@ -200,10 +200,16 @@ impl Neg for Zr {
 pub struct G1([u8; G1_SIZE_FR256]);
 
 impl G1 {
-    pub fn new() -> G1 {
+    /// Create a new point which binary representation consists of all zeros.
+    pub fn null() -> G1 {
```

Comment:
may be use `zero` name like in other crypto types ?
```suggestion
    pub fn zero() -> G1 {
```

Reply:
I don't like both variants. zero() can be perceived as zero point on the curve... I think that this problem is caused by bad design of block header. Why we want to store "empty" signature and then change it?